packaging/debian/copyright

```diff
@@ -0,0 +1,35 @@
+Upstream Homepage:
+    https://www.gen2vdr.de/wirbel/w_scan/index2.html
```

Comment:
I'm your upstream, so this should be "https://github.com/stefantalpalaru/w_scan2".